python_modules/dagster/dagster/core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -232,11 +232,6 @@ def store_event(self, event):
             conn.execute(insert_event_statement)
 
         if event.is_dagster_event and event.dagster_event.asset_key:
-            check.invariant(
-                event.dagster_event_type == DagsterEventType.ASSET_MATERIALIZATION
-                or event.dagster_event_type == DagsterEventType.ASSET_OBSERVATION,
-                "Can only store asset materializations and observations in index database",
-            )
```

Comment:
why remove instead of update?

Reply:
no strong reason here...I didn't feel like the check was necessary, but I've readded and updated the check now